src/GameController/GCExtendedGamepadSnapshot.cs

```diff
@@ -63,6 +60,56 @@ public NSData ToNSData ()
 			return p == IntPtr.Zero ? null : new NSData (p);
 		}
 	}
+	
+	// float_t are 4 bytes (at least for ARM64)
+	[StructLayout (LayoutKind.Sequential, Pack = 1)]
+	[iOS (7,0)]
+	[Mac (10,9, onlyOn64: true)]
+	public struct GCExtendedGamepadSnapshotData {
+
+		// Standard information
+		public ushort /* uint16_t */ Version;
+		public ushort /* uint16_t */ Size;
+
+		// Extended gamepad data
+		// Axes in the range [-1.0, 1.0]
+		public float /* float_t = float */ DPadX;
+		public float /* float_t = float */ DPadY;
+
+		// Buttons in the range [0.0, 1.0]
+		public float /* float_t = float */ ButtonA;
+		public float /* float_t = float */ ButtonB;
+		public float /* float_t = float */ ButtonX;
+		public float /* float_t = float */ ButtonY;
+		public float /* float_t = float */ LeftShoulder;
+		public float /* float_t = float */ RightShoulder;
+
+		// Axes in the range [-1.0, 1.0]
+		public float /* float_t = float */ LeftThumbstickX;
+		public float /* float_t = float */ LeftThumbstickY;
+		public float /* float_t = float */ RightThumbstickX;
+		public float /* float_t = float */ RightThumbstickY;
+
+		// Buttons in the range [0.0, 1.0]
+		public float /* float_t = float */ LeftTrigger;
+		public float /* float_t = float */ RightTrigger;
+
+		[TV (12, 2), Mac (10, 14, 4, onlyOn64: true), iOS (12, 2)]
+		bool LeftThumbstickButton;
+		
+		[TV (12, 2), Mac (10, 14, 4, onlyOn64: true), iOS (12, 2)]
+		bool RightThumbstickButton;
+
+		[DllImport (Constants.GameControllerLibrary)]
+		static extern /* NSData * __nullable */ IntPtr NSDataFromGCExtendedGamepadSnapshotData (
+			/* GCExtendedGamepadSnapshotData * __nullable */ ref GCExtendedGamepadSnapshotData snapshotData);
```

Comment:
that API did not exists before Xcode 10.2 right ?
so it's missing it's availability